### PR TITLE
I have add the icons of some apps in Belgium we use here

### PR DESCRIPTION
**Pull request by ruditimmermans**

### CODE REVIEW

**Comment by ruditimmermans** [comment]
Adding correct named icon and remove the wrong one, and adding the Belgium apps into Appgrid

**Comment by wurzer** [comment]
Some icons have not completely in white.

**Comment by ruditimmermans** [comment]
Ok i check them again sorry

File: AppGrid.qml [hunk 1]
```diff
@@ -78,13 +95,30 @@ Page {
...
+        "com.bookmark.money": qsTr("Money Lover"),
+        "com.lscsmartconnection.smart": qsTr("LSC Smart Connect"),
+        "be.bmid.itsme": qsTr("Itsme"),
+        "be.engie.smart": qsTr("Engie Smart"),
+        "be.ixor.doccle.android": qsTr("Doccle"),
+        "com.codesynd.cashfree": qsTr("Bonsai"),
+        "com.asvcorp.aftershock": qsTr("AfterShock"),
+        "be.tweedehands.m": qsTr("2dehands"),
+        "be.argenta.bankieren": qsTr("Argenta"),
+        "be.belgacom.hello": qsTr("MyProximus"),
```
Comment (Roker2) [inline AppGrid.qml:112]: If this apps has same names, I don't understand for why you added this code. Maybe better to remove it.

**Comment by ruditimmermans** [comment]
Not all app names have same name but i will remove the ones has the same name thanks!

**Comment by ruditimmermans** [comment]
@wurzer The icons should now all be into the correct white color if i'm not mistake...

Thanks!

**Pull request merged**